CONTRIBUTING.md

```diff
@@ -112,6 +112,23 @@ Both are run automatically with the following command:
 ```
 
 
+## Dependencies
+
+Python dependencies are managed with [Pipenv](https://docs.pipenv.org/).
+Required packages are handled in [Pipfile](Pipfile) with the fully
+specified dependencies in `Pipfile.lock`. When changing requirements,
+run `pipenv lock` to update the lock file. The `--keep-outdated` option
+can be used to keep existing pins from being updated.
+
+The documentation is also built on [Read the
+Docs](https://readthedocs.org/), but that service doesn't support
+Pipenv. Dependencies for Read the Docs are managed in the
+[requirements.readthedocs.txt](requirements.readthedocs.txt) file, which
+is generated from `Pipfile.lock` by running `pipenv lock --requirements
+--dev > requirements.readthedocs.txt`. Please update this file when
+updating `Pipfile.lock`.
```

Comment:
I'm not quite sure where in the GitHub Actions workflow to put this, but: presumably this output is deterministic, since that's kind of the point of locking the dependencies. So we can check in CI that:

```
diff -u requirements.readthedocs.txt <(pipenv lock --requirements --dev)
```

exits with status 0, or I guess,

```
pipenv lock --requirements --dev > requirements.readthedocs.txt
git diff --exit-code requirements.readthedocs.txt
```

Reply:
I meant to say this in the commit message and forgot. I explicitly ran this with `--keep-outdated` since I didn't want to pull in a bunch of updates randomly. That said, we should probably do that going forward. I just didn't want to roll that into the same work. I'm going to punt on that for now but I opened #168 to handle that in the future.

Reply:
Oh, I think I focused on the wrong thing there. You just want to make sure that someone did actually updated `requirements.readthedocs.txt` in CI. I think that can be done. Let me try to put that in.

Reply:
I added something now that seems to work.